src/imap_mag/appConfig.py

```diff
@@ -18,17 +19,32 @@ class Source(BaseModel):
 class Destination(BaseModel):
     folder: Path = Path(".")
     filename: str
+    export_to_database: bool = True
 
 
 class PacketDefinition(BaseModel):
     hk: Path
 
 
+class API(BaseModel):
+    webpoda_url: Optional[str] = None
+    sdc_url: Optional[str] = None
+
+
 class AppConfig(BaseModel):
     source: Source
     work_folder: Path = Path(".work")
     destination: Destination
     packet_definition: Optional[PacketDefinition] = None
+    api: Optional[API] = None
+
+    def __init__(self, **kwargs):
+        kwargs = dict((key.replace("_", "-"), value) for (key, value) in kwargs.items())
```

Comment:
```suggestion
		# replace hypens with underscores so that you can build config from constructor args, and still have them mapped to the hyphen split property names in the YAML files
        kwargs = dict((key.replace("_", "-"), value) for (key, value) in kwargs.items())
```